src/components/Identity/getEcidCommand/createIdentityPayload.js

```diff
@@ -0,0 +1,18 @@
+import createRequestPayload from "../../../core/edgeNetwork/requestPayloads/createRequestPayload";
+
+export default () => {
+  return createRequestPayload(content => {
+    return {
+      addIdentity: (namespaceCode, identity) => {
+        content.identityMap = content.identityMap || {};
+        content.identityMap[namespaceCode] =
+          content.identityMap[namespaceCode] || [];
+        content.identityMap[namespaceCode].push(identity);
```

Comment:
I see this was duplicated from `core/edgeNetwork/requestPayloads/createDataCollectionRequestPayload.js`. I think it would be good to create a module at `core/edgeNetwork/requestPayloads/contentModifiers/addIdentityFactory.js`. The module would export a function that could be used from both here and `core/edgeNetwork/requestPayloads/createDataCollectionRequestPayload.js` like this:

`addIdentity: createAddIdentity(content),`